workflows/setup-workflows.yml

```diff
@@ -30,14 +30,11 @@ jobs:
       - name: Copy workflow files
         run: |
           rsync -a --include='*/' --include='*.yml' --exclude='*' ./ckeditor4-workflows-common/workflows/ ./.github/workflows/
+          rm -rf ckeditor4-workflows-common
           if [[ $(git status --porcelain) ]]; then
               echo "HAS_CHANGES=1" >> $GITHUB_ENV
           fi
```

Comment:
I think we can indeed go with `git status .github/workflows/ --procelain`.

As for cleaning... yes I was thinking about it, since VM is simply removed after workflows is done. OTOH, it's good to keep things clean so further modifying and working with this workflow is more predictable?

Reply:
Sure, it won't hurt cleaning after yourself.